tests/tests_app_examples/custom_work_dependencies/app.py

```diff
@@ -24,7 +24,7 @@ class WorkWithCustomBaseImage(LightningWork):
     def __init__(self, cloud_compute: CloudCompute = CloudCompute(), **kwargs):
         # this image has been created from ghcr.io/gridai/base-images:v1.8-cpu
         # by just adding an empty file at /content/.e2e_test
-        image_tag = os.getenv("LIGHTNING_E2E_TEST_IMAGE_VERSION", "v0.0.1")
+        image_tag = os.getenv("LIGHTNING_E2E_TEST_IMAGE_VERSION", "v1.12")
         custom_image = f"ghcr.io/gridai/image-for-testing-custom-images-in-e2e:{image_tag}"
```

Comment:
@Borda Where can I find the dockerfile of `ghcr.io/gridai/image-for-testing-custom-images-in-e2e`?